px-map-behavior-layer-IMS.es6.js

```diff
@@ -176,9 +182,12 @@
     createInst(options) {
       const defaultMarkerIcon = '<svg xmlns="http://www.w3.org/2000/svg" version="1.1"  height="16" width="16"><circle cx="8" cy="8" r="6" stroke="#3E87E8" stroke-width="3" fill="#88BDE6" fill-opacity="0.4"/></svg>';
       const defaultMarkerIconURL = "data:image/svg+xml;base64," + btoa(defaultMarkerIcon);
+      const customPaneName =  options.pane.name || options.layerName;
 
       //Create a custom pane to draw onto so that we can control the draw order.
-      this.parentNode.elementInst.createPane(options.pane || options.layerName);
+      this.parentNode.elementInst.createPane(customPaneName);
```

Comment:
I guess it would be good practice to set this.parentNode.elementInst to a variable so we don't have to look it up 3 times.